### ISSUE TITLE
All radio and checkbox inputs should have labels or other descriptive text 

### ISSUE THREAD

**Issue by blnkt** [ManifoldScholar/manifold#1103]
Screen Readers expect form controls to each have their own associated label.  If no label is found it will use any adjacent text element as its description (in the case of most inputs it is looking first at the text element immediately preceding it, in the case of checkboxes and radios it will first look immediately after.  Attributes such as aria-label can be used on the form control itself to provide a description, or aria-labelledby and aria-describedby can be used in conjunction with unique ids to provide descriptions from disparate text elements.  These strategies are the most widely supported and reliable, but as suggested above a text element in the vicinity may also suffice but should be tested.
